src/applications/check-in/components/AppointmentBlock.jsx

```diff
@@ -36,9 +36,14 @@ const AppointmentBlock = props => {
   return (
     <div>
       {app === APP_NAMES.CHECK_IN && (
-        <p className="vads-u-font-family--serif" data-testid="date-text">
+        <h2 className="vads-u-font-family--serif" data-testid="date-text">
```

Comment:
I'm confused by this change for the day-of app. This should probably just get removed since this component isn't used in day-of

Reply:
I think this gets used on the details page of day of.

Reply:
The only usage I can see doing a codebase search is in `<PreCheckinConfirmation>` I don't see this h2 in the wires either